docs/src/api/class-request.md

```diff
@@ -62,21 +62,24 @@ Returns the [Frame] that initiated this request.
 **DEPRECATED** Incomplete list of headers as seen by the rendering engine. Use [`method: Request.allHeaders`] instead.
 
 ## async method: Request.headersArray
-* langs: js, csharp, python
-- returns: <[Array]<[Array]<[string]>>>
-
-An array with all the request HTTP headers associated with this request. Unlike [`method: Request.allHeaders`], header names are not lower-cased.
-Headers with multiple entries, such as `Set-Cookie`, appear in the array multiple times.
-
-## async method: Request.headersArray
-* langs: java
 - returns: <[Array]<[Object]>>
   - `name` <[string]> Name of the header.
   - `value` <[string]> Value of the header.
 
-An array with all the request HTTP headers associated with this request. Unlike [`method: Request.allHeaders`], header names are not lower-cased.
+An array with all the request HTTP headers associated with this request. Unlike [`method: Request.allHeaders`], header names are NOT lower-cased.
 Headers with multiple entries, such as `Set-Cookie`, appear in the array multiple times.
 
+## async method: Request.headerValue
```

Comment:
Let's add `Request.headerValues` too.

Reply:
It'll never return multiple values, so we can add it later.